contribute/README.md

```diff
@@ -10,9 +10,13 @@ Thanks for enhancing OpenFisca anyway!
 
 ## Why contribute to OpenFisca?
 
-OpenFisca is a project being developed under the GPLv3 license or later.
+OpenFisca is a project being developed under the [AGPL-3.0](https://www.gnu.org/licenses/agpl-3.0.en.html) license.
 The source code is freely available and modifiable.
 
+Therefore, if you improve or update modelled tax-benefit systems (country packages, extensions, reforms, etc.) or the core engine, we ask you to share them back, for it allows the community to grow :)
+
+Although, as long as the software you build with OpenFisca can't be considered an improvement or an evolution of OpenFisca, we won't consider it a form of derivative work. If the latter holds true, we won't ever ask you to share your software's source code, whether you're using the Python API, the Web API or both.
```

Comment:
I really cannot back this phrasing. I find it raises more questions and opens more loopholes than it gives certainty.

I see the value of your goal. I believe this should be made of archetypical examples that make the threat model explicit (e.g. a startup that helps companies assess their business plan by making computations and showing them in some proprietary software is ok; an accounting company that adds a set of rules for a specific sector and doesn't share it back is a big no, and this is why).

We can pair on that if you want 🙂

Reply:
I agree language is unfortunate. Having the goal clear, I think we can iterate 😃 .

Reply:
@MattiSG How do you prefer to proceed? I had just these ideas in mind:

- Pushing and re-asking for comments (drawback: comments will be hidden)
- Here in comments (drawback: comments will be hidden before merge)
- In pull request's comments (preferred, though drawback: not granular enough)
- In a EvilDoc, FramaDoc, etc. (best user experience, but can't capitalise discussion)

Reply:
Push and re-ask, we'll just open a new discussion linking to previous discussions if necessary.